subjects/PE-x86/VCExeSample/VCExeSample.h

```diff
@@ -373,44 +395,49 @@ typedef struct Globals {
 	real32 r4020E4;	// 4020E4
 	real32 r4020E8;	// 4020E8
 	real32 r4020EC;	// 4020EC
-	Eq_55 gbl_c;	// 403018
+	Eq_59 gbl_c;	// 403018
 } Eq_1;
 
-typedef void (Eq_5)(char *, int32, char *, real32);
+typedef void (Eq_5)(char *, int32, char *, Eq_10);
+
+typedef union Eq_10 {
+	real32 u0;
+	real64 u1;
+} Eq_10;
```

Comment:
Yeah, this union is caused by the printf format string interpreter coercing the real32 into a real64.